src/Sulu/Bundle/TestBundle/Resources/dist/sulu.yml

```diff
@@ -54,13 +54,26 @@ sulu_core:
         en: English
     translations: ['de', 'en']
     fallback_locale: 'en'
-    phpcr:
-        backend:
-            type: "%phpcr.transport%"
-            url:  "%phpcr.backend_url%"
-        workspace: "%phpcr.workspace%"
-        username: "%phpcr.username%"
-        password: "%phpcr.password%"
+
+doctrine_phpcr:
+    session:
+        sessions:
+            default:
+                backend:
+                    type: "%phpcr.transport%"
+                    url:  "%phpcr.backend_url%"
+                    check_login_on_server: false
+                workspace: "%phpcr.workspace%"
+                username: "%phpcr.username%"
+                password: "%phpcr.password%"
+            live:
+                backend:
+                    type: "%phpcr.transport%"
+                    url:  "%phpcr.backend_url%"
+                    check_login_on_server: false
+                workspace: live
+                username: "%phpcr.username%"
+                password: "%phpcr.password%"
```

Comment:
Move this to the `sulu_document_manager` configuration? Automate it based on configured managers?